lambdas/constants.js

```diff
@@ -44,7 +44,12 @@ const ReadOnlyPolicy = `{
               "ce:GetCostAndUsage",
               "redshift-serverless:List*",
               "lambda:GetCodeSigningConfig",
-              "lambda:GetFunctionCodeSigningConfig"
+              "lambda:GetFunctionCodeSigningConfig",
```

Comment:
This was supposed to be sorted.
Not sure how we broke it.
Can you sort the list?

Reply:
sorted